models/aeif_cond_alpha.nestml

```diff
@@ -119,7 +119,7 @@ neuron aeif_cond_alpha_neuron:
       emit_spike()
     end
 
-    I_stim = currents.get_sum()
+    I_stim = currents.get_sum()*1pA
```

Comment:
I don't understand this change (see our conversation on #328 and my recent comment on the same problem in [PR343](https://github.com/nest/nestml/pull/343#pullrequestreview-21196362))